react/features/recording/components/Recording/native/RecordingConsentDialog.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import Dialog from 'react-native-dialog';
+
+/**
+ * Component that renders the dialog for explicit consent for recordings.
+ *
+ * @returns {JSX.Element}
+ */
+export default function RecordingConsentDialog() {
+    const { t } = useTranslation();
+    const [ visible, setVisible ] = useState(true);
+
+    const closeDialog = useCallback(() => {
+        setVisible(false);
+    }, []);
+
+    return (
+        <Dialog.Container
+            coverScreen = { false }
+            visible = { visible }>
+            <Dialog.Title>{ t('dialog.recordingInProgressTitle') }</Dialog.Title>
+            <Dialog.Description>
+                { t('dialog.recordingInProgressDescription') }
+            </Dialog.Description>
+            <Dialog.Button
```

Comment:
IIRC we do have our own wrapper. @Calinteodor ?

Reply:
ConfirmDialog